src/rocker/templates/cuda_dev_snippet.Dockerfile.em

```diff
@@ -0,0 +1,23 @@
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    gnupg2 curl ca-certificates && \
+    apt-key del 7fa2af80 && \
+    curl -L -O https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-keyring_1.0-1_all.deb && \
+    dpkg -i cuda-keyring_1.0-1_all.deb && \
+    sudo sed -i '/developer\.download\.nvidia\.com\/compute\/cuda\/repos/d' /etc/apt/sources.list && \
+    apt-get purge --autoremove -y curl \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    cuda nvidia-cuda-dev \
```

Comment:
I've made changes to use the package manager to install necessary packages without defining specific versions. I've tested it inside WSL. I have not tested with a generic Ubuntu system but it should work.